pkg/pillar/types/global.go

```diff
@@ -1030,6 +1038,11 @@ func NewConfigItemSpecMap() ConfigItemSpecMap {
 	configItemSpecMap.AddStringItem(FmlCustomResolution, FmlResolutionUnset, blankValidator)
 	configItemSpecMap.AddStringItem(TUIMonitorLogLevel, "info", blankValidator)
 
+	// Log deduplication and filtering settings
+	configItemSpecMap.AddIntItem(LogDedupWindowSize, 100, 0, 0xFFFFFFFF)
```

Comment:
I think the defaults should reflect the current behaviour so that no unexpected changes to the users are coming. But maybe it's not that critical in this case...

Reply:
that's an excellent point! I thought it makes sense to enable deduplication by default, but it comes with a performance penalty - so I'll disable it until we have discussed it further